Makefile

```diff
@@ -69,79 +49,23 @@ bdddeps:
-		docker buildx inspect multibuilder --bootstrap;\
-		docker buildx use multibuilder;\
-	fi
-
-.PHONY: build  
-build: go-build docker.buildx docker-build
-
-go-build:
-	@echo "------------------"
-	@echo "--> Build Chaos Exporter"
-	@echo "------------------"
-	@bash build/go-multiarch-build.sh ./cmd/exporter 
-
-docker-build: 
-	@echo "------------------"
-	@echo "--> Build chaos-exporter image" 
-	@echo "------------------"
-	# Dockerfile available in the repo root
-	@docker buildx build --file Dockerfile --progress plane --platform linux/arm64,linux/amd64 --no-cache --tag $(DOCKER_REPO)/$(DOCKER_IMAGE):$(DOCKER_TAG) .
-
 .PHONY: test
 test:
 	@echo "------------------"
 	@echo "--> Run Go Test"
 	@echo "------------------"
 	@go test ./... -v -count=1
 
-.PHONY: security-checks
-security-checks: trivy-security-check
-
-trivy-security-check:
-	@echo "------------------"
-	@echo "--> Trivy Security Check"
-	@echo "------------------"
-	./trivy --exit-code 0 --severity HIGH --no-progress $(DOCKER_REPO)/$(DOCKER_IMAGE):$(DOCKER_TAG)
-	./trivy --exit-code 1 --severity CRITICAL --no-progress $(DOCKER_REPO)/$(DOCKER_IMAGE):$(DOCKER_TAG)
-
-.PHONY: push
-push: docker.buildx docker-push
-
-docker-push:
-	@echo "------------------"
-	@echo "--> Push chaos-exporter image" 
-	@echo "------------------"
-	REPONAME="$(DOCKER_REPO)" IMGNAME="$(DOCKER_IMAGE)" IMGTAG="$(DOCKER_TAG)" ./buildscripts/push
+.PHONY: build-chaos-exporter build-chaos-exporter-amd64 push-chaos-exporter
 
-unused-package-check:
-	@echo "------------------"
-	@echo "--> Check unused packages for the chaos-operator"
-	@echo "------------------"
-	@tidy=$$(go mod tidy); \
-	if [ -n "$${tidy}" ]; then \
-		echo "go mod tidy checking failed!"; echo "$${tidy}"; echo; \
-	fi
+publish-chaos-exporter: build-chaos-exporter push-chaos-exporter
 
-.PHONY: build-amd64
-build-amd64:
-	@echo "-------------------------"
-	@echo "--> Build chaos-exporter image" 
-	@echo "-------------------------"
-	@sudo docker build --file Dockerfile --tag $(DOCKER_REPO)/$(DOCKER_IMAGE):$(DOCKER_TAG) . --build-arg TARGETARCH=amd64
+build-chaos-exporter:
+	@docker buildx build --file Dockerfile --progress plane --platform linux/arm64,linux/amd64 --tag $(DOCKER_REPO)/$(DOCKER_IMAGE):$(DOCKER_TAG) .
 
-.PHONY: push-amd64
-push-amd64:
+build-chaos-exporter-amd64:
+	@docker build -f Dockerfile -t $(DOCKER_REPO)/$(DOCKER_IMAGE):$(DOCKER_TAG) .  --build-arg TARGETPLATFORM="linux/amd64"
 
-	@echo "------------------------------"
-	@echo "--> Pushing image" 
-	@echo "------------------------------"
-	@sudo docker push $(DOCKER_REPO)/$(DOCKER_IMAGE):$(DOCKER_TAG)	
\ No newline at end of file
+push-chaos-exporter:
+	@docker buildx build --file Dockerfile --progress plane --push --platform linux/arm64,linux/amd64 --tag $(DOCKER_REPO)/$(DOCKER_IMAGE):$(DOCKER_TAG) .
+	@docker buildx build --file Dockerfile --progress plane --push --platform linux/arm64,linux/amd64 --tag $(DOCKER_REPO)/$(DOCKER_IMAGE):latest .
```

Comment:
This will update the `latest` image tag for every push?

Reply:
removed the latest image tag. now `latest` tag is coming as an env from github script